backend/src/drivers/sqlite.ts

```diff
@@ -21,6 +21,11 @@ const initSchema = async (db: Database) => {
   await db.run(
     "CREATE TABLE IF NOT EXISTS transactionResults (cid TEXT PRIMARY KEY, transaction_result TEXT, head_cid TEXT)"
   );
+
+  // Object Ownership
+  await db.exec(
+    "CREATE TABLE IF NOT EXISTS object_ownership (cid TEXT, user_id TEXT, is_admin BOOLEAN, marked_as_deleted BOOLEAN, PRIMARY KEY (cid, user_id))"
```

Comment:
I'm not sure if I see the utility of having `is_admin` directly in the object_ownership
First, this property seems to never be use anywhere
Second, user role should be tied to the user object, and not the content they uploaded.

If today I'm an admin, there is chance that at a later date, I lost that role and become a simple user, that will be weird to still have files flag as admin and could lead to logic error

Reply:
I think we could improve on `marked_as_deleted` to use a dated [soft delete approach](https://hasura.io/docs/2.0/schema/common-patterns/data-modeling/soft-deletes/) so a `deleted_at` and while we are at it, I believe adding a `created_at` (and for a table that could be edited `updated_at`)

Reply:
Makes sense

Reply:
The updated_at, created_at I'll add it when I migrate DB to postgres